### ISSUE TITLE
Runs are exposed across contexts via the WES API 

### ISSUE THREAD

**Issue by patmagee** [aws/amazon-genomics-cli#503]
[//]: #  (Please do not include your AWS account or credentials as part of the bug request.)

**Describe the Bug**

If a user has two MiniWDL contexts deployed in the same project, they can access the same set of runs via the `/ga4hg/wes/v1/runs/{run_id}` endpoint regardless of what context they use. It does not appear that the runs are bound to a given context. The run list endpoint works appropriately and only shows runs that have been submitted to the individual context

[//]: #  (Please describe the behavior that you are seeing with as many details as possible including logs from AGC, CloudWatch snippets and CloudFormation errors if relevant.)

**Steps to Reproduce**

1. Create a new `MiniWDL` context (Not sure if this bug applies to other engines)
2. Run a workflow and record the `run_id` 
3. Create a new MiniWDL context
5. Using the WES API for the new context, visit the `GET /ga4gh/wes/v1/{run_id}` where the run_id was from the previous run
6. 
**Expected Behavior**
The API to return a `404` status error indicating that the run does not exist 

**Actual Behavior**
The Old `RunLog` from the previous context is returned

**Screenshots**

![Peek 2022-08-16 16-38](https://user-images.githubusercontent.com/8960037/184980592-561de768-e82a-437e-af32-663540ddcd69.gif)

**Additional Context**

I am using the WES api directly

**Comment by markjschreiber** [comment]
I agree, each context should have it's own APIGateway wes endpoint and it's own WES Adapter and that WES Adapter should not return information for runs that are not part of the same context.
